pymeasure/instruments/thorlabs/__init__.py

```diff
@@ -23,3 +23,4 @@
 #
 
 from .thorlabspm100usb import ThorlabsPM100USB
+from .thorlabspro8000 import thorlabsPro8000
```

Comment:
Please also add the [documentation stubs](https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html#adding-documentation), and add yourself to AUTHORS.txt

Reply:
I did it in my fork. I am new to github and I don't know if I should do something else ?

Reply:
If you have it in your master branch and push this to your github repo, it will appear here.
Note in general it's good practice to use a separate branch (not your master) for pull requests, but it's not an issue now.

Reply:
So you can see it now ? I just have to click "Resolve Conversation" ?

Reply:
yes, everything except the authors.txt change. You can also look at https://github.com/ralph-group/pymeasure/pull/291/files

Reply:
Done.

Reply:
I don't see an updated authors.txt. Did you push that already?

Reply:
Should be ok this time. I am sorry I am total newbie to github.

Reply:
No need, we all started at one time. All good now!